### DISCUSSION TITLE
Sharing UF2 Files

### DISCUSSION THREAD

**Discussion by ZX-80** [earlephilhower/arduino-pico#830] [Q&A] [answered]
I was going to upload some UF2 files as part of a GitHub release, since it makes updates easy for the users, and while quickly searching through one in a hex editor, I noticed it gave two full paths:

- C:\Users\REDACTED\AppData\Local\Arduino15\packages\rp2040\hardware\rp2040\1.13.1\libraries\SDFS\src\SDFS.h

- c:\users\REDACTED\appdata\local\arduino15\packages\rp2040\hardware\rp2040\1.13.1\pico-sdk\src\common\pico_base\include\pico/types.h

Is there a way to remove these when generating the UF2? Anything else I should worry about?

**Comment by earlephilhower** [comment, marked as answer]
They're just strings and not going to introduce any dependencies on those paths, of course.  Not sure why you're worried, but it's not a big deal to remove them.

They come from `assert()`s in the code.  Use Tools->Debug Level->NDEBUG and rebuild and they should go away.
